Dockerfile

```diff
@@ -31,8 +31,11 @@ ENV PATH "$PATH:/root/.yarn/bin:/root/.config/yarn/global/node_modules/.bin"
 COPY Gemfile Gemfile.lock .ruby-version ./
 RUN ${BUNDLE_INSTALL_CMD}
 
+CMD ["bundle", "exec", "rails", "server"]
+
+
+FROM base as production
```

Comment:
Does aliasing this have a performance impact?  Excuse my ignorance here.

Reply:
Looks like it just loads the previous layer while building, so there will be no meaningful impact